src/main.rs

```diff
@@ -140,6 +140,43 @@ fn install_hook() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+/// Copy the text te the clipboard.
+///
+/// This function never returns on Linux and various BSDs. See [here][1] for more info.
```

Comment:
I read the `wait` method docs but I don't see why this method wouldn't return or why it exits the process itself.